code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -967,3 +967,37 @@ var/list/datum/dna/hivemind_bank = list()
 	feedback_add_details("changeling_powers","ED")
 	return 1
 
+/mob/proc/changeling_armblade()
+	set category = "Changeling"
+	set name = "Generate Arm Blade (20)"
+	set desc="Transform one of our arms into a deadly blade."
+
+	if(!istype(src, /mob/living/carbon/human)) return
```

Comment:
OK I know this is a port so it's not really the best place to ask this, but if only `/mob/living/carbon/human`s can use it, why is it a `/mob`-level proc?